translations/pt-BR/README.md

```diff
@@ -0,0 +1,250 @@
+```
+
+com a seguinte instrução:
+
+```json
+"VARIANT": "3.9-bullseye"
+```
+
+Essa alteração instrui o Codespaces a usar o Python 3.9 em vez do 3.8.
+
+Se você fizer qualquer alteração na configuração do `devcontainer.json`, uma caixa aparecerá após salvar.
+
+![Recriando Codespace](./images/Codespace_rebuild.png)
+
+Clique em reconstruir. Aguarde até que seu Codespace reconstrua o ambiente VS Code.
+
+### Etapa 2: adicionar uma extensão
+
+Seu ambiente vem com extensões pré-instaladas. Você pode alterar com quais extensões seu ambiente Codespaces começa. Veja como:
+
+1. Abra o arquivo [.devcontainer/devcontainer.json](./.devcontainer/devcontainer.json) e localize o seguinte elemento JSON
+**extensões**:
+
+   ```json
+   "extensions": [
+    "ms-python.python",
+    "ms-python.vscode-pylance"
+   ]
+   ```
+
+1. Adicione _"ms-python.black-formatter"_ à lista de extensões. Ele deve acabar se parecendo com o seguinte:
+
+   ```json
+   "extensions": [
+    "ms-python.python",
+    "ms-python.vscode-pylance",
+    "ms-python.black-formatter"
+   ]
+   ```
+
+   Essa string é o identificador exclusivo do [Black Formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter&WT.mc_id=academic-77460-alfredodeza), uma extensão popular para formatar código Python de acordo com as melhores práticas. Adicionar o identificador _"ms-python.black-formatter"_ à lista permite que os Codespaces saibam que essa extensão deve ser pré-instalada na inicialização.
+
+   Aviso: Ao alterar qualquer configuração no json, aparecerá uma caixa após salvar.
+
+   ![Recriando o Codespace](../../images/Codespace_rebuild.png)
+
+   Clique em reconstruir. Aguarde seu codespace reconstruir o ambiente do VS Code.
+
+Para encontrar o identificador exclusivo de uma extensão:
+
+- Navegue até a página da extensão, por exemplo [https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter](https://marketplace.visualstudio.com/items?itemName=ms- python.black-formatter&WT.mc_id=academic-77460-alfredodeza)
+- Localize o campo *Identificador exclusivo* na seção **Mais informações** à sua direita.
+
+## 🤖 Use o Copilot para criar uma aula para os alunos
+O GitHub Copilot agora está disponível no GitHub Codespaces. Você pode usar o Copilot para criar uma aula para seus alunos. Este repositório inclui a extensão para Copilot para que você possa usá-lo imediatamente. Certifique-se de que sua conta tenha acesso ao Copilot. Caso não tenha acesso, você pode [solicitar acesso aqui](https://github.com/login?return_to=%2Fgithub-copilot%2Fsignup).
+
+O GitHub Copilot é gratuito para alunos e professores. [Saiba mais](https://education.github.com/pack/offers). Siga [estas etapas](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba- p/3736279?WT.mc_id=academic-0000-alfredodeza) para verificar sua associação de aluno ou professor e habilitar o Copilot gratuitamente.
```

Comment:
link do artigo: https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-setting-up-github-student-and-github-copilot-as-an/ba-p/3736279?WT.mc_id=academic-0000-alfredodeza